1.6.1/rockcraft.yaml

```diff
@@ -28,6 +28,3 @@ parts:
       install -D -m755 pushgateway ${CRAFT_PART_INSTALL}/bin/pushgateway
     organize:
       pushgateway: bin/pushgateway
-  prometheus-pushgateway:
-    source-tag: v1.6.1
-    build-snaps: go/1.19/stable
```

Comment:
Is this something CI added?

Reply:
Yes, I filled and issue: https://github.com/canonical/observability/issues/103